### ISSUE TITLE
Conflicts With Bootstrap Drowdown

### ISSUE THREAD

**Issue by VermontMediaSolutions** [clineamb/bootstrap-offcanvas#17]
This conflicts with the Bootstrap Drop down ability, so only a single layered navigation is possible.

**Comment by clineamb** [comment]
@VermontMediaSolutions Can I see some sample code?

**Comment by extricate** [comment]
Has this bug been fixed yet? If no, is a fix scheduled?

**Comment by clineamb** [comment]
No, has not, since I never got a response.  Can look into soon.

**Comment by extricate** [comment]
Ah alright, thanks for the reply. I can confirm I've been experiencing the unfortunate effects of this bug as well.

**Comment by dafreels** [comment]
I added this to my application css and it resolved the issue:

.drawer ul {
    display: none;
}

The problem seems to be with the 'display: inherit' being set by the drawer.
